corehq/apps/reports/static/reports/javascripts/maps_utils.js

```diff
@@ -403,6 +403,9 @@ function makeDisplayContext(metric, setActiveFeature) {
             // store visibility on the feature object so datatables can access it
             feature.visible = (feature._conf != null);
             if (!feature.visible) {
+                if(feature.$tr === void(0)){
```

Comment:
also could this just be changed to `if(feature.$tr)`? it seems like if it was `null` the next line would also fail.